packages/editor-ui/src/Interface.ts

```diff
@@ -477,6 +483,10 @@ export interface IPushDataExecutionFinished {
 	retryOf?: string;
 }
 
+export interface IPushDataNodeDescriptionUpdated {
+	data: {};
+}
```

Comment:
There is no payload, so let's remove this? Else let's convey `nodeType` or other useful info in the payload - `nodeType` could be useful if in the future we want to reopen the NDV for the node being worked on.